db/migrate/20210930125828_add_reason_to_ecf_participant_eligibilities.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddReasonToECFParticipantEligibilities < ActiveRecord::Migration[6.1]
+  def change
+    add_column :ecf_participant_eligibilities, :reason, :string, null: false, default: "none"
```

Comment:
Is this reason for the participant not being eligible?

If so, can we call it `ineligible_reason`? `reason_ineligible`?

Reply:
Not quite. I think it covers any status reason.

Reply:
I guess it's the reason for the status, there aren't any reasons for `eligible` but you'd have to put something in hence the `none` reason.